mchf-eclipse/drivers/ui/ui_driver.c

```diff
@@ -4516,12 +4516,18 @@ static void UiDriver_HandleTXMeters()
         swrm.p_curr   = 0;
         swrm.fwd_calc = 0;
         swrm.rev_calc = 0;
+        swrm.high_vswr_detected = 0;
```

Comment:
if you change it to line below it's more obvious that this variable is type of bool, isn't it?
`
swrm.high_vswr_detected = false;
`